runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -594,6 +596,11 @@ private void takeScreenshot(String fileName, String subDir, Image image)
 		imageCapture.takeScreenshot(screenshot, fileName, subDir, config.notifyWhenTaken(), config.uploadScreenshot());
 	}
 
+	private boolean isInsideGauntlet()
+    {
+		return this.client.isInInstancedRegion() && this.client.getMapRegions().length > 0 && (this.client.getMapRegions()[0] == GAUNTLET_REGION_ID || this.client.getMapRegions()[0] == CORRUPTED_GAUNTLET_REGION_ID);
```

Comment:
This needs to account for instances
```suggestion
              final Player localPlayer = client.getLocalPlayer();
		if (localPlayer == null || !client.isInInstancedRegion())
		{
			return false;
		}
		final WorldPoint p = WorldPoint.fromLocalInstance(client, localPlayer.getLocalLocation());
		return p.getRegionID() == GAUNTLET_REGION_ID || p.getRegionID() == CORRUPTED_GAUNTLET_REGION_ID;
```

Reply:
`return this.client.isInInstancedRegion()`

Reply:
Break these conditions onto separate lines.
```suggestion
		return this.client.isInInstancedRegion()
			&& this.client.getMapRegions().length > 0
			&& (this.client.getMapRegions()[0] == GAUNTLET_REGION_ID
				|| this.client.getMapRegions()[0] == CORRUPTED_GAUNTLET_REGION_ID);
```